packages/x-tree-view/src/TreeItem/TreeItem.tsx

```diff
@@ -2,14 +2,15 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import Collapse from '@mui/material/Collapse';
-import { resolveComponentProps, useSlotProps } from '@mui/base/utils';
 import useForkRef from '@mui/utils/useForkRef';
 import { shouldForwardProp } from '@mui/system/createStyled';
 import { alpha } from '@mui/material/styles';
 import { TransitionProps } from '@mui/material/transitions';
+import { unstable_composeClasses as composeClasses } from '@mui/utils';
```

Comment:
To unify import style. 👍 
```suggestion
import composeClasses from '@mui/utils/composeClasses';
```